step-00/android/app/.classpath

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
uh oh, there's too many files in this RP. We need to .gitignore everything under build/

Reply:
Fixed in https://github.com/googlecodelabs/flutter-cupertino-store/pull/7

The .gitignore changes I needed indicate that the .gitignore that `flutter create` is generating is slightly wrong. Any hints on where the flutter create template is hiding?

Reply:
I think I found it: https://github.com/flutter/flutter/pull/30023